src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -241,6 +241,7 @@ protected override bool DoTryComplete(SocketAsyncContext context)
         private sealed class ReceiveMessageFromOperation : ReadOperation
         {
             public byte[] Buffer;
+            public IList<ArraySegment<byte>> Buffers;
```

Comment:
The other operation classes (ReceiveOperation, SendOperation) list this member at the end, after Flags.  We should be consistent.